airbyte-integrations/connectors/destination-e2e-test/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: unknown
   connectorType: destination
   definitionId: 2eb65e87-983a-4fd7-b3e3-9d9dc6eb8537
```

Comment:
Just to clarify, if we drop the cloud version but don't change the definitionId or name of the oss version, then we effectively create a new cloud connector with a new name? That is what we want? All existing connections using dev-null will just be parked at the old version until they switch over?